quill.js

```diff
@@ -28,6 +28,7 @@ import CodeBlock, { Code as InlineCode } from './formats/code';
 import Formula from './modules/formula';
 import Syntax from './modules/syntax';
 import Toolbar from './modules/toolbar';
+import Authorship from './modules/authorship';
```

Comment:
The other modules are imported in a alphabetical order; so this should be at the top.